third-party-libs/index.md

```diff
@@ -1,7 +1,12 @@
-# Module content
+# Third-party libraries
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+## Table of contents
+
+1. Finding libraries
+2. Determining library compatibility
+3. Installing and linking libraries
+   1. Android
+   2. IOS
+4. Adding icons
```

Comment:
what do you think about "Adding custom fonts"?

Reply:
I suppose this is very important topic for beginners and sometimes it's hard to implement for them.